supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -11,5 +11,28 @@
 [[admonitions]]
 == Admonitions
 
+Admonitions should draw the reader’s attention to certain information. Keep admonitions to a minimum, and avoid placing multiple admonitions in close proximity to one another. If multiple admonitions are necessary, restructure the information by removing the less important statements and placing them into the flow of the main content.
+
+Allowed admonition types:
+
+- NOTE
+- IMPORTANT
+- WARNING
+- TIP
+
+CAUTION, which is another type of AsciiDoc admonition, is not fully supported by the portal. This admonition should not be used.
```

Comment:
We should probably say "by the Red Hat Customer Portal" instead of "the portal", since this might eventually be used by teams outside of Red Hat.

Reply:
Also, not critical, but this note about the CAUTION admonition would probably fit nicely into a NOTE or IMPORTANT admonition :smile:

Reply:
I like that. 👍